src/utils.js

```diff
@@ -31,4 +31,33 @@ export function findBetween(content, left, right) {
     return '';
   }
   return content.substring(start, end);
+}
+
+function normalDensity(mean, scale, x) {
```

Comment:
Since this function is only used within `normalDistributionStages` I suggest declaring it in the scope of the function, rather than globally. 

I also added a few missing semicolons, removed a few unnecessary semicolons, and renamed a few variables to make the code more clear. 

```js
export function normalDistributionStages(maxVus, durationSeconds, numberOfStages=10) {
  function normalDensity(mean, scale, x) {
    return Math.exp(-1 / 2 * Math.pow((x - mean) / scale, 2)) / (scale * Math.sqrt(2 * Math.PI));
  }

  const mean = 0;
  const scale = 1;
  let curve = new Array(numberOfStages + 2).fill(0);
  let durations = new Array(numberOfStages + 2).fill(Math.ceil(durationSeconds / 6));
  let k6stages = [];

  for (let i = 0; i <= numberOfStages; i++) {
    curve[i] = normalDensity(mean, scale, -2 * scale + 4 * scale * i / numberOfStages);
  }

  let peakDistribution = Math.max(...curve);

  let vus = curve.map(x => Math.round(x * maxVus / peakDistribution));

  for (let j = 1; j <= numberOfStages; j++) {
    durations[j] = Math.ceil(4 * durationSeconds / (6 * numberOfStages));
  }

  for (let k = 0; k <= numberOfStages + 1; k++) {
    k6stages.push({duration: `${durations[k]}s`, target: vus[k]});
  }

  return k6stages;
}

```

If you agree with these suggestions, please update the code. 

Since I know that you are also planning on adding "Poisson distribution", it might be a good idea to do it inside of the same PR, in case you want to reuse some of the same code. If you want to do it in separate PRs, that's also OK.

Reply:
Changes added. :)
I prefer to do Poisson in a separate PR.